src/UserForm.js

```diff
@@ -75,6 +79,7 @@ class UserForm extends React.Component {
         label={<FormattedMessage id="ui-requests.requester.pickupServicePoint" />}
         component={Select}
         fullWidth
+        validate={this.requireServicepoint}
```

Comment:
if a selection is required, shouldn't we pass `required` to its `<Field>` as well?

also, minor, `requireServicePoint` is a more consistent name for this function.